src/model/scripts/pytorch_lighting_training/pl_configurations/checkpointing/default_checkpointing.yaml

```diff
@@ -0,0 +1,3 @@
+save_dir: /path/to/save_dir
```

Comment:
Can we make 2 files
- `default_checkpointing.example.yaml` - this file commits to git
- `default_checkpointing.yaml` - this file added in `.gitignore`
This will help prevent someone accidentally commit their new config file into our git